Bottom: e3522e79dcd82adbff601e20f4ef1a978b359d32
Top:    498152a704d464975f8918850896c5dd0a3d43d0
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-22 19:52:08 -0400

Refresh of egg-support

---

diff --git a/include/configs/rw_egg.h b/include/configs/rw_egg.h
index 18f0804..e51d5e8 100644
--- a/include/configs/rw_egg.h
+++ b/include/configs/rw_egg.h
@@ -171,7 +171,7 @@
 "loadaddr=0x31000000\0" \
 "rd_addr=0x32000000\0" \
 "usbtty=cdc_acm\0" \
-"ramargs=setenv bootargs console=ttyS0,115200n8 root=/dev/mmcblk0p3 init=/etc/preinit rw loglevel=7\0" \
+"ramargs=setenv bootargs console=ttyS0,115200n8 root=/dev/mmcblk0p3 rootwait init=/etc/preinit rw loglevel=7\0" \
 "serverip=192.168.1.48\0" \
 "nfsargs=setenv bootargs console=ttyS0,115200n8 root=/dev/nfs init=/etc/preinit rw nfsroot=${serverip}:${rootpath} ip=dhcp loglevel=7\0" \
 "rootpath=/tftpboot/arm\0" \
